scripts/build.js

```diff
@@ -0,0 +1,52 @@
+#! /usr/bin/env node
+/* eslint-disable no-console */
+
+const cp = require('child_process');
+const fs = require('fs-extra');
+const path = require('path');
+const copyExt = require('./copy-ext');
+
+const args = process.argv.slice(2);
+const buildExt = args.indexOf('--ext') !== -1;
+const buildCore = args.indexOf('--core') !== -1;
+const mode = args[args.indexOf('--mode')] || 'production';
+const watch = args[args.indexOf('-w')];
+
+// cleanup old build
+fs.removeSync(path.resolve(process.cwd(), 'dist'));
+fs.removeSync(path.resolve(process.cwd(), 'core/esm'));
+
+function exec(cmd, stdio = 'inherit') {
```

Comment:
consider useing the javascript api instead
(`const build = require('@nebula.js/cli-build')`)

Reply:
okey.. I can switch to that... could be better, yes.

Reply:
done